assets/javascripts/caseListFiltersClient.js

```diff
@@ -1,9 +0,0 @@
-/* eslint-disable */
-
-$(document).ready(init)
-
-function init() {
-  $('#licenceFilters').change(function() {
-    window.location = $(this).val()
-  })
-}
```

Comment:
removed as causing a vulnerability and not being used anywhere (no reference anywhere else of id `licenceFilters`)